acceptance: TestCompose failed

acceptance.TestCompose [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_Acceptance/15338755?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_Acceptance/15338755?buildTab=artifacts#/) on master @ [d9dbc2ef1e9db9491e7af7530d148df85b4d6214](https://github.com/cockroachdb/cockroach/commits/d9dbc2ef1e9db9491e7af7530d148df85b4d6214):


```
=== RUN   TestCompose
--- FAIL: TestCompose (83.49s)
=== RUN   TestCompose/ComposeGSSFlyway
--no-ansi option is deprecated and will be removed in future versions. Use ``--ansi never`` instead.
using --exit-code-from implies --abort-on-container-exit
Creating network "flyway_default" with the default driver
Pulling flyway (flyway/flyway:9.3)...
Head "https://registry-1.docker.io/v2/flyway/flyway/manifests/9.3": Get "https://auth.docker.io/token?scope=repository%3Aflyway%2Fflyway%3Apull&service=registry.docker.io": EOF
    compose_test.go:94: --no-ansi option is deprecated and will be removed in future versions. Use ``--ansi never`` instead.
        using --exit-code-from implies --abort-on-container-exit
        Creating network "flyway_default" with the default driver
        Pulling flyway (flyway/flyway:9.3)...
        Head "https://registry-1.docker.io/v2/flyway/flyway/manifests/9.3": Get "https://auth.docker.io/token?scope=repository%3Aflyway%2Fflyway%3Apull&service=registry.docker.io": EOF
        
    compose_test.go:95: exit status 1
    --- FAIL: TestCompose/ComposeGSSFlyway (10.54s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/sql-foundations
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestCompose.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

acceptance.TestCompose [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_Acceptance/15392682?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_Acceptance/15392682?buildTab=artifacts#/) on master @ [526fc7fdb09bde93bfcf94ee7b1dd333e0a3ce8a](https://github.com/cockroachdb/cockroach/commits/526fc7fdb09bde93bfcf94ee7b1dd333e0a3ce8a):


```
=== RUN   TestCompose
--- FAIL: TestCompose (54.95s)
=== RUN   TestCompose/ComposeGSSPython
--no-ansi option is deprecated and will be removed in future versions. Use ``--ansi never`` instead.
using --exit-code-from implies --abort-on-container-exit
Found orphan containers (gss_psql_1) for this project. If you removed or renamed this service in your compose file, you can run this command with the --remove-orphans flag to clean it up.
Pulling python (us-east1-docker.pkg.dev/crl-ci-images/cockroach/acceptance-gss-python:20221214-141947)...
Get "https://us-east1-docker.pkg.dev/v2/": dial tcp: lookup us-east1-docker.pkg.dev on 127.0.0.53:53: server misbehaving
    compose_test.go:94: --no-ansi option is deprecated and will be removed in future versions. Use ``--ansi never`` instead.
        using --exit-code-from implies --abort-on-container-exit
        Found orphan containers (gss_psql_1) for this project. If you removed or renamed this service in your compose file, you can run this command with the --remove-orphans flag to clean it up.
        Pulling python (us-east1-docker.pkg.dev/crl-ci-images/cockroach/acceptance-gss-python:20221214-141947)...
        Get "https://us-east1-docker.pkg.dev/v2/": dial tcp: lookup us-east1-docker.pkg.dev on 127.0.0.53:53: server misbehaving
        
    compose_test.go:95: exit status 1
    --- FAIL: TestCompose/ComposeGSSPython (0.39s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestCompose.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

--- a/pkg/acceptance/compose/flyway/docker-compose.yml
+++ b/pkg/acceptance/compose/flyway/docker-compose.yml
@@ -10,7 +10,7 @@ services:
   flyway:
     depends_on:
       - cockroach
-    image: flyway/flyway:9.3
+    image: us-east1-docker.pkg.dev/crl-docker-sync/docker-io/flyway/flyway:9.3
     volumes:
       - ./sql:/sql
     command: migrate -user=root -url=jdbc:postgresql://cockroach:26257/defaultdb -locations=filesystem:/sql

